.eslintrc

```diff
@@ -5,6 +5,7 @@
       "ignoreComments": true,
       "ignoreUrls": true,
       "ignorePattern": "^\\s*var\\s.+=\\s*(require\\s*\\()|(/)"
-    }]
+    }],
+    "no-extra-semi": "error"
```

Comment:
Please move this configuration to https://github.com/strongloop/eslint-config-loopback so that it's eventually enabled for all LB (3.x) projects/repositories.